test/stack.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I believe we are currently going in the direction of using a `behavior.yaml` first. Though, I can't tell if this is used by any automated tests, so maybe it doesn't affect anything

Reply:
It doesn't really affect anything, purely for local testing purposes. But I think we can use them in test in the future.

As per the name, this controller doesn't know any file name conventions; it just `GET`s the `StackDefinition` object from the k8s cluster. I just changed the name to `stackdefinition.yaml` to be more consistent with its actual content.